### DISCUSSION TITLE
No updates on Kubic project for podman 3.4.x?

### DISCUSSION THREAD

**Discussion by zyrikby** [containers/podman#12265] [Q&A] [answered]
I have just checked yesterday and found out that the latest version of podman is currently 3.4.1. However, the latest update on the Kubic project is 3.3.1 (I use Ubuntu), which is 2 months old. I do not see updates there for most recent podman versions. Therefore, I have a question: is the update procedure has been changed, and if yes then where can I find new instructions (the install guide on podman.io contains the outdated instructions)?

**Comment by Luap99** [comment]
@lsm5 PTAL

**Comment by afbjorklund** [comment]
@zyrikby : This is normal, it takes a couple of weeks for the Mac version to be updated and a couple of months for the Kubic version.

There are "testing" versions available in other channels (both for CoreOS and for Ubuntu), so that you can test pre-release versions:

https://builds.coreos.fedoraproject.org/browser?stream=testing-devel

https://build.opensuse.org/package/show/devel:kubic:libcontainers:testing/podman

But there is no support for these...

You can also build it, from source ?

**Reply by zyrikby** [reply]
Really? I think before I received the updates on Kubic much faster than now. However, I might be wrong.

Yes, I can build it from the sources. However, I prefer to use software installed with your package manager, so that I do not need to care about building and installing all the updates.

Let's wait what @lsm5 will reply. As I see he is the official maintainer of the Kubic repository.

**Reply by afbjorklund** [reply]
I think the packages come faster to Fedora, than to the other distributions ?

Debian has their own packaging (from source), with a different update rate.

**Comment by lsm5** [comment]
i'll update podman on kubic

**Comment by lsm5** [comment, marked as answer]
sent podman 3.4.2 update to kubic repo just now. It should hopefully be available to install in an hour or less.

RE: update schedule, well, i have a lot of other things on my plate these days. Also, the build process for ubuntu packages involves me rebasing the debian packaging branch on top of the latest published tag. Quite often this rebase doesn't go smoothly, so automation breaks. Fedora's packaging doesn't need to mess with the source repo the way debian's packaging does, so the automation for that doesn't break that easily.

Anyway, I hope to be able to provide a smooth transition to the default Ubuntu packages once 22.04 LTS is released, after which there will be no more Ubuntu updates on Kubic.

**Reply by afbjorklund** [reply]
For reference, the Ubuntu packages were using podman 3.2.1 when Kubic packages were using podman 3.3.1.

So this would mean going back to a slower pace, like the Debian packages that currently use 3.0.1 for "stable"

(or the RHEL packages)

Thanks for the update!

**Reply by afbjorklund** [reply]
Rootless podman stopped working after upgrade (#12374), but `sudo podman` still worked. Something about fuse-overlayfs.
